src/Analyzers/CSharp/Tests/AddBraces/AddBracesTests.cs

```diff
@@ -448,9 +466,10 @@ class Program
 {
     static void Main()
     {
-{}
```

Comment:
this changed because there's a parent (empty) `if` statement that was found that we added teh braces to.  This is not at all bodacious.  Better fix is the new behavior which the code change caused us to have.